app/views/content_items/_body_with_related_links.html.erb

```diff
@@ -1,9 +1,13 @@
 <% content_for :simple_header, true %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <%= render 'govuk_publishing_components/components/title',
-      title: @content_item.title %>
+  <div class="govuk-grid-column-two-thirds responsive-top-margin">
```

Comment:
A lot of the problems with spacing introduced by switching to the heading component can be fixed by adding the `govuk-main-wrapper` class to the `main` element in the application layout. It's something we've not added for some reason but is supposed to be part of doing Design System layout. Apologies, I should have made you aware of this earlier.

You'll probably need to do something extra to override this class in some instances (for example on HTML publication pages, where the extra spacing isn't needed) but should save having to manually add spacing to each individual page type.